src/core/listeners/handlers/dragenter.handler.js

```diff
@@ -1,3 +1,23 @@
 export const dragenterHandler = function (e) {
   this.related = e.target;
+
+  if (typeof this.defaultOptions.onDragenter === 'function') {
+    try {
+      this.defaultOptions.onDragenter(
+        {
+          nativeEvent: e,
+          stop: () => {
+            throw new Error(`Stop method is available only for callbacks
+                    'onDragstart' and 'onDragend'. For more info look at
+                    https://github.com/Vivify-Ideas/vue-draggable/blob/master/README.md
+                  `);
```

Comment:
Indentation is wrong, but we can ignore it for now... 
I'll probably add husky precommit lint hook